src/WCSimTrackingAction.cc

```diff
@@ -18,6 +18,9 @@ WCSimTrackingAction::WCSimTrackingAction()
 
   ProcessList.insert("Decay") ;                         // Michel e- from pi+ and mu+
   ProcessList.insert("conv") ;                         // Products of gamma conversion
+  ProcessList.insert("phot");
```

Comment:
Will the file size be increased a lot with e.g. usual electron/muon gun simulation?